packages/espressocash_app/lib/features/tokens/data/token_dao.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:drift/drift.dart';
+import '../../../data/db/db.dart';
+
+part 'token_dao.g.dart';
+
+@DriftAccessor(tables: [TokenRows])
+class TokenDao extends DatabaseAccessor<MyDatabase> with _$TokenDaoMixin {
```

Comment:
I don't see value in this class, this logic can be sitting directly in the repository.